standards/css.md

```diff
@@ -1,4 +1,4 @@
-# CSS & Less Guide
+# CSS & Less Standards
```

Comment:
Minor thing, but in the README this page is listed as "CSS/Less" and here it's "CSS & Less". Personally I prefer the latter but it'd be good if they were consistent.